tests/test_self_chat.py

```diff
@@ -29,3 +29,16 @@
             ]
         )
         self_chat.self_chat(opt)
+
+    def test_ed(self):
+        pp = self_chat.setup_args()
+        opt = pp.parse_args(
+            [
+                '-mf',
+                'zoo:blender/blender_3B/model',
+                '-t',
+                'empathetic_dialogues',
+                '--seed-messages-from-task',
+            ]
+        )
+        self_chat.self_chat(opt)
```

Comment:
Can we use SelfChat.main(task='empathetic_dialogues', ...)

Reply:
Why or what would be the difference to use this?

Reply:
Cleaner and newer syntax. Functionally equivalent.